### PR TITLE
Document that proc named fooTask is created for every foo task [backport]

### PR DESCRIPTION
**Pull request by kaushalmodi**
Ref: https://irclogs.nim-lang.org/01-05-2020.html#15:18:03

/cc @genotrance

### CODE REVIEW

**Comment by kaushalmodi** [comment]
@narimiran Looks like an unrelated failure

https://builds.sr.ht/~araq/job/199613#task-test-827

```
Error: execution of an external program failed: '/usr/home/build/Nim/testament/testament --pedantic all -d:nimCoroutines'
FAILURE
+ nim c -r tools/ci_testresults.nim
```

File: lib/system/nimscript.nim [hunk 1]
```diff
@@ -379,11 +379,26 @@ when not defined(nimble):
...
     ##  task build, "default build is via the C backend":
     ##    setCommand "c"
+    ##
+    ## For a task named ``foo``, this template generates a ``proc`` named
+    ## ``fooTask``.  This is useful if you need to call one task in
+    ## another in your Nimscript.
+    ##
+    ## Example:
+    ##
+    ## .. code-block:: nim      # > nim foo
```
Comment (Araq) [inline lib/system/nimscript.nim]: So you commented on the ``.. code-block`` itself? Come on, did you even check how this renders in the HTML?

Comment (kaushalmodi) [reply lib/system/nimscript.nim]: No .. let me check. ~A comment breaks that? Is that a bug?~

🤦‍♂️ Oh, I'll fix it.

Comment (kaushalmodi) [reply lib/system/nimscript.nim]: Now, fixed and confirmed as well.

image

**Pull request merged**